sdk/go/auto/local_workspace.go

```diff
@@ -381,6 +381,15 @@ func (l *LocalWorkspace) InstallPlugin(ctx context.Context, name string, version
 	return nil
 }
 
+// InstallPluginFromServer acquires the plugin matching the specified name and version from a third party server.
+func (l *LocalWorkspace) InstallPluginFromServer(ctx context.Context, name string, version string, server string) error {
+	stdout, stderr, errCode, err := l.runPulumiCmdSync(ctx, "plugin", "install", "resource", name, version, "--server", server)
```

Comment:
I noticed that "resource" was the default and hardcoded here. Thoughts?

Reply:
I think that's fine, you could probably simplify your worries about default parameters in the other SDKs by just hard coding them to "resource" as well. No one installs languages or analyzers yet, and by the time we've got good support for those we'll probably be overhauling automation api as well.

Reply:
I've refactored this so that `resource` is the default across most SDKs